google-cloud-redis/lib/google/cloud/redis/credentials.rb

```diff
@@ -17,14 +17,10 @@
 module Google
   module Cloud
     module Redis
-      class Credentials < Google::Auth::Credentials
-        SCOPE = [
-          "https://www.googleapis.com/auth/cloud-platform"
-        ].freeze
-        PATH_ENV_VARS = %w(REDIS_KEYFILE GOOGLE_CLOUD_KEYFILE GCLOUD_KEYFILE)
-        JSON_ENV_VARS = %w(REDIS_KEYFILE_JSON GOOGLE_CLOUD_KEYFILE_JSON GCLOUD_KEYFILE_JSON)
-        DEFAULT_PATHS = ["~/.config/gcloud/application_default_credentials.json"]
-      end
+      ##
+      # @deprecated Use version-specific credentials classes
+      #
+      Credentials = Google::Cloud::Redis::V1beta1::Credentials
```

Comment:
This gem isn't GA, so we can remove this file instead of deprecating it.

Reply:
Removed.